src/state/Data/AttributeData/actions.js

```diff
@@ -216,63 +268,131 @@ function getIndexData(spatialData, attributeData) {
 }
 
 // ============ actions ============
-function addDataAction(key, data) {
+function removeSpatialIndexAction(filter, order) {
```

Comment:
Keep established naming - actionRemoveSpatialIndex, actionAddDataAndIndex, etc.

Reply:
Fixed: 51fa7d16990cd09eeeec1efd7e11c831abadedff

Reply:
Well, that's just the one, but it's a start :)

Reply:
hih. 5fe7b6ef75bdfd7e182613bc4958bf74a78014d6 checked in all actions in data/**/actions.js